VKappProject/Network/VKService.swift

```diff
@@ -0,0 +1,49 @@
+// VKService.swift
+// Copyright © RoadMap. All rights reserved.
+
+//  Created by Александр Андреевич Щепелин on 22.11.2022.
+
+import Alamofire
+import Foundation
+
+/// Сетевой слой
+final class VKService {
+    // MARK: - Public Method
+
+    func loadFriendList() {
+        let url = Constants.URLComponents.baseURL + Constants.URLComponents.friendsMethod + Constants.URLComponents
```

Comment:
```suggestion
        let url = Constants.URLComponents.baseURL + Constants.URLComponents.friendsMethod + Constants.URLComponents
```
делай через URLComponents всегда